src/wp-content/themes/twentytwentyone/assets/css/ie-editor.css

```diff
@@ -1306,6 +1306,17 @@ pre.wp-block-preformatted {
 	color: currentColor;
 	border-color: currentColor;
 	position: relative;
+	font-size: 2rem;
+	font-style: normal;
+	font-weight: 700;
+	letter-spacing: normal;
+}
+
+@media only screen and (min-width: 652px) {
+
+	.wp-block-pullquote {
+		font-size: 2rem;
+	}
```

Comment:
Why do we need a similar `font-size` CSS in the media query? We already have the same size in line 1309.

Reply:
It's not pretty, but that comes from the build process. 
`.wp-block-pullquote p` had a similar media query in the compiled Internet Explorer stylesheets before this patch.

The `postcss` process seems to round the value up because other browsers would calculate the variable to 1.875rem (1.25 * 1.5rem) at smaller screen sizes and 2rem for larger screens.